### ISSUE TITLE
Failed to train tensorflow object detection on priviate detection dataset

### ISSUE THREAD

**Issue by guanlinting** [tensorflow/models#2782]
Hi, I want to train tensorflow object detection framework on my dataset, so I create record with my dataset, and it can train on ssd framework (for both mobilenet and inception v2), but it cannot train on faster rcnn framework, I have tried for resnet50, resnet101 and faster_rcnn_inception_resnet_v2_atrous, it all report the follow error:

>INFO:tensorflow:Starting Queues.
INFO:tensorflow:Error reported to Coordinator: <class 'tensorflow.python.framework.errors_impl.DataLossError'>, corrupted record at 0
	 [[Node: parallel_read/ReaderReadV2 = ReaderReadV2[_device="/job:localhost/replica:0/task:0/cpu:0"](parallel_read/TFRecordReaderV2, parallel_read/filenames)]]
INFO:tensorflow:global_step/sec: 0
INFO:tensorflow:Caught OutOfRangeError. Stopping Training.

I just fighting it for one day, and just cannot find the problem. I use tensorflow1.4 and python 2.7
below is the function for creating code modified from create_pet_tf_record.py
`def create_tf_example(img_path,  data,
                       label_map_dict,
                       image_subdirectory,
                       ignore_difficult_instances=False):
  if not os.path.exists(img_path):
    raise ValueError('{} is not exists!'.format(img_path))
  with tf.gfile.GFile(img_path, 'rb') as fid:
    encoded = fid.read()
  image = PIL.Image.open(img_path)
  if image.format != 'PNG':
    raise ValueError('Image format not PNG')
  image_format = 'png'.encode('utf8') 

  width = image.size[0]
  height = image.size[1]

  xmin = []
  ymin = []
  xmax = []
  ymax = []
  classes = []
  classes_text = []

  for obj in data:
    xmin.append(float(obj[1]) / width)
    ymin.append(float(obj[2]) / height)
    xmax.append(float(obj[3]) / width)
    ymax.append(float(obj[4]) / height)
    class_name = str('vehicle')
    classes_text.append(class_name.encode('utf8'))
    classes.append(1)

  example = tf.train.Example(features=tf.train.Features(feature={
      'image/height': dataset_util.int64_feature(height),
      'image/width': dataset_util.int64_feature(width),
      'image/filename': dataset_util.bytes_feature(
          img_path.encode('utf8')),
      'image/source_id': dataset_util.bytes_feature(
          img_path.encode('utf8')),
      'image/encoded': dataset_util.bytes_feature(encoded),
      'image/format': dataset_util.bytes_feature(image_format),
      'image/object/bbox/xmin': dataset_util.float_list_feature(xmin),
      'image/object/bbox/xmax': dataset_util.float_list_feature(xmax),
      'image/object/bbox/ymin': dataset_util.float_list_feature(ymin),
      'image/object/bbox/ymax': dataset_util.float_list_feature(ymax),
      'image/object/class/text': dataset_util.bytes_list_feature(classes_text),
      'image/object/class/label': dataset_util.int64_list_feature(classes)
  }))
  return example`

**Comment by guanlinting** [comment]
It's my fault, i gives wrong record file to pipeline config file
